Gradle configuration fails because of implicit dependency on ShadowJar

No template, so I will just freestyle the issue report to the best of my abilities!

OS: Manjaro Linux 18.1.4 / Kernel 5.4.2 amd64
Java: 1.8.0_232
Gradle: 6.0.1 (wrapper via `gradlew`)

## Expected behaviour
When following the steps on the Quickstart guide in the `README.md` file, I can incorporate the Gradle plugin `id("io.kotless")` into my existing build without breaking the project.

## Actual behaviour
Upon following the instructions on how to include Kotless in my own Gradle project, Gradle fails during configuration stage with the following error message:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/suushie_maniac/jvdocs/tnoodle/webscrambles/build.gradle.kts' line: 27

* What went wrong:
An exception occurred applying plugin request [id: 'io.kotless', version: '0.1.2']
> Failed to apply plugin [id 'io.kotless']
   > Task with name 'shadowJar' not found in project ':webscrambles'.
```

The failure occurs even for simple commands such as `./gradlew tasks`.

In particular, my buildscript file uses the Kotlin DSL. I have installed Kotless as `id("io.kotless").version("0.1.2")` as determined by the Quickstart guide. The `plugins {}` block also includes `johnrengelman`'s `shadow` plugin indenpendently of Kotless, because it is needed as part of my project.

## Steps to reproduce
The entire codebase of the project where this fail occurs is open-source at https://github.com/suushiemaniac/tnoodle/tree/feature/kotless.

To reproduce, simply clone and check out the `feature/kotless` branch that is linked above and execute `./gradlew tasks` in the project directory.

The relevant build file is in the `cloudscrambles` folder, found at `cloudscrambles/build.gradle.kts`

@TanVD I just started to play around. I copy/pasted the build.gradle.kts from the README.md. Started it in a new empty folder. Here the dump.

Just asking for a (tmp) workaround, would still like to try out kotless.

```
/tmp/xxx$ cat build.gradle.kts 
plugins {
    id("io.kotless") version "0.1.2" apply true
    
    //Version of Kotlin should 1.3.50+
    kotlin("jvm") version "1.3.50" apply true
}

repositories {
    jcenter()
}

dependencies {
    implementation("io.kotless", "lang", "0.1.2")
    //or for Ktor
    //implementation("io.kotless", "ktor-lang", "0.1.2")
}

kotless {
    config {
        bucket = "kotless.s3.example.com"
        
        dsl {
            type = DSLType.Kotless
            //or for Ktor
            //type = DSLType.Ktor
        }

        terraform {
            profile = "example"
            region = "us-east-1"
        }
    }

    webapp {
        route53 = Route53("kotless", "example.com")

        //configuration of lambda created
        lambda {            
            //needed only for Kotless DSL
            kotless {
                //Define packages in which scan for routes should be performed
                packages = setOf("io.kotless.examples")
            }
        }
    }
}

/tmp/xxx$ gradle
FAILURE: Build failed with an exception.

* Where:
Build file '/tmp/xxx/build.gradle.kts' line: 1

* What went wrong:
An exception occurred applying plugin request [id: 'io.kotless', version: '0.1.2']
> Failed to apply plugin [id 'io.kotless']
   > Task with name 'shadowJar' not found in root project 'xxx'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s

/tmp/xxx$ gradle --version

------------------------------------------------------------
Gradle 6.0.1
------------------------------------------------------------

Build time:   2019-11-18 20:25:01 UTC
Revision:     fad121066a68c4701acd362daf4287a7c309a0f5

Kotlin:       1.3.50
Groovy:       2.5.8
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          1.8.0_192 (Azul Systems, Inc. 25.192-b01)
OS:           Linux 4.15.0-72-generic amd64
```

--- a/plugins/gradle/src/main/kotlin/io/kotless/plugin/gradle/utils/Utils.kt
+++ b/plugins/gradle/src/main/kotlin/io/kotless/plugin/gradle/utils/Utils.kt
@@ -5,7 +5,5 @@ import org.gradle.kotlin.dsl.apply
 
 /** Apply a plugin if it is not already applied. */
 internal fun Project.applyPluginSafely(id: String) {
-    if (!plugins.hasPlugin(id)) {
-        apply(plugin = id)
-    }
+    pluginManager.apply(id)
 }

